RELEASE_NOTES.md

```diff
@@ -21,6 +21,10 @@ v1.0.0-alpha.xx
   opt-in for manager implementations.
   [#163](https://github.com/OpenAssetIO/OpenAssetIO/issues/163)
 
+- Migrated `getWithRelationship` and `getWithRelationships` to C++
+  with Python bindings.
+  [#913](https://github.com/OpenAssetIO/OpenAssetIO/issues/913)
```

Comment:
[fix] As these methods didn't exist before this release, can we just mention that they're now in C++ in the breaking change note above?

Reply:
Combined in c77a8e35.  I've assumed that the four "Migrate" commits will be squashed together (otherwise will need a few more fixup commits).

Reply:
Spot on 👍 thanks.